server/controllers/groups.js

```diff
@@ -309,13 +309,10 @@ export const create = (req, res, next) => {
       })
     })
     .tap(g => {
-      // if there is a host id provided, we add the collective to the host
-      if (group.HostId) {
-        return User.findOne({ where: { id: group.HostId }}).tap(h => {
-          host = h;
-          _addUserToGroup(g, host, {role: roles.HOST, remoteUser: creator})
-        })
-      }
+      return User.findOne({ where: { id: group.HostId || defaultHostId() }}).tap(h => {
```

Comment:
That means we'll always be setting a host, regardless of the mission of the group?